kornia/augmentation/_2d/geometric/resize.py

```diff
@@ -44,7 +43,9 @@ def compute_transformation(self, input: Tensor, params: Dict[str, Tensor], flags
         if params["output_size"] == input.shape[-2:]:
             return eye_like(3, input)
 
-        transform: Tensor = get_perspective_transform(params["src"], params["dst"])
+        transform: Tensor = torch.as_tensor(
```

Comment:
Why as tensor needed here ?

Reply:
Since the src and dst are long type, we need to do some conversions here.